tests/global_functions.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * @author  Andreas Heigl<andreas@heigl.org>
+ * @license http://www.wtfpl.net/txt/copying/ WTFPL
+ * @since   08.02.15
+ */
+function emptyFunc()
+{
+    // Empty Function as required by codesniffer instead of an empty anonymous
+    // function
+}
```

Comment:
I don't think it is really necessary to bootstrap the tests with this function. A `function () {}` goes perfectly through my phpcs. For yours it doesn't? Which version do you use?